### PR TITLE
Change path from /article/title-of-story to /articles/title-of-story

### PR DESCRIPTION
**Pull request by marineb**

### CODE REVIEW

**Comment by illusionOfParadise** [comment]
To change `articles/tout` to `articles`, you should add default category to `actualitesAction` in `ArticleController`:
defaults={**"category"="tout"**, "page"=1, "_enable_campaign_silence"=true}

File: src/Controller/EnMarche/ArticleController.php [hunk 1]
```diff
@@ -19,13 +20,17 @@ class ArticleController extends Controller
...
      *     "/articles/{category}/{page}",
      *     requirements={"category"="\w+", "page"="\d+"},
-     *     defaults={"page"=1, "_enable_campaign_silence"=true},
+     *     defaults={"category"="tout", "page"=1, "_enable_campaign_silence"=true},
      *     name="articles_list"
      * )
      * @Method("GET")
      */
-    public function actualitesAction(string $category, int $page): Response
+    public function actualitesAction(string $category, int $page, Request $request): Response
```
Comment (Remg) [inline src/Controller/EnMarche/ArticleController.php]: Please specify the `$request` parameter in first position

File: src/Redirection/DynamicRedirectionsSubscriber.php [hunk 2]
```diff
@@ -44,7 +54,7 @@ public static function getSubscribedEvents()
 
     public function onKernelException(GetResponseForExceptionEvent $event)
     {
-        if (!$event->getException() instanceof NotFoundHttpException) {
+        if (!$event->getException() instanceof NotFoundHttpException || !$event->getException()) {
```
Comment (Remg) [inline src/Redirection/DynamicRedirectionsSubscriber.php]: if (!$event->getException() || !$event->getException() instanceof NotFoundHttpException) {

File: src/Redirection/DynamicRedirectionsSubscriber.php [hunk 3]
```diff
@@ -63,13 +73,33 @@ public function onKernelException(GetResponseForExceptionEvent $event)
...
                 && isset($routeParams['uuid'])
                 && ($eventEntity = $this->eventRepository->findOneByUuid($routeParams['uuid']))
                 && !$eventEntity->isPublished()) {
                 $redirectCode = Response::HTTP_FOUND;
             }
 
+            if (0 === strpos($requestUri, '/article/')) {
+                $pathToContent = substr($requestUri, 9);
+                $urlToRedirect = null !== ($article = $this->articleRepository->findOnePublishedBySlug($pathToContent))
+                    ? $urlToRedirect = $this->router->generate('article_view', [
```
Comment (Remg) [inline src/Redirection/DynamicRedirectionsSubscriber.php]: you can remove `$urlToRedirect =` here

**Pull request merged**